.github/workflows/ci.yml

```diff
@@ -49,6 +49,8 @@ jobs:
 
       - run: yarn install
 
+      - run: yarn lint:js
```

Comment:
Can we run this once instead of for every supported Ember version?

Reply:
it's matching what we were already doing before, but generally we could, yes. I just wanted to keep the PR relatively small for now.

Reply:
It seems before this we weren't running `lint` at all on CI though?

Reply:
we were running it via ember-cli-eslint, which generates Mocha tests that were run for each ember-try scenario

Reply:
ah I see – ok 👍